src/peer_select.cc

```diff
@@ -1152,9 +1157,8 @@ PeerSelector::interestedInitiator()
 
 bool
 PeerSelector::wantsMoreDestinations() const {
-    const auto maxCount = Config.forward_max_tries;
-    return maxCount >= 0 && foundPaths <
-           static_cast<std::make_unsigned<decltype(maxCount)>::type>(maxCount);
+    int maxCount = Config.forward_max_tries;
```

Comment:
If you can undo changes to this specific line, please do so.
```suggestion
    const auto maxCount = Config.forward_max_tries;
```

Reply:
It can be undone. Undoing it.